betfair/models.py

```diff
@@ -138,7 +138,7 @@ class Match(BetfairModel):
 
 
 class Runner(BetfairModel):
-    selection_id = Field(DataType(float), required=True)
+    selection_id = Field(DataType(six.text_type), required=True)
```

Comment:
This might be a python specific thing. I was using 2.7.9 (on windows via cygwin) and it was generating unicode strings instead of floats. I reverted the change in my branch (see link below) but i was making a lot of changes at the time - this might be worth re testing.

https://github.com/goetzk/betfair.py/commit/8fbc19e59e49a8d6574e51d7fa7e0cc138718dbc